Content.Shared/_RMC14/Xenonids/Rest/XenoRestBindInputSystem.cs

```diff
@@ -0,0 +1,33 @@
+using Content.Shared._RMC14.Input;
+using Robust.Shared.Input.Binding;
+
+namespace Content.Shared._RMC14.Xenonids.Rest;
+
+public sealed class XenoRestBindInputSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        CommandBinds.Builder
+            .Bind(CMKeyFunctions.RMCRest,
+                InputCmdHandler.FromDelegate(session =>
+                    {
+                        var ent = session?.AttachedEntity;
+                        if (ent != null && HasComp<XenoComponent>(ent.Value))
+                        {
+                            var ev = new XenoRestActionEvent();
```

Comment:
You can't raise an action event like this, it will have none of the data that it usually does, like the performer, action id, etc
It also will not check for cooldowns
You need to find the action on the xeno and then call ActionsSystem.TriggerAction